test/thirdparty.test.ts

```diff
@@ -0,0 +1,227 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
This is pretty cool. I like having this here because when we update these deps, we can make sure our lib is still compatible. Also, a great example of how to use these others libs with Tableland.